genvm/templates/multisig/types.go

```diff
@@ -0,0 +1,39 @@
+package multisig
+
+import "github.com/spacemeshos/go-spacemesh/genvm/core"
+
+//go:generate scalegen
+
+// SpawnArguments contains a collection with PublicKeys.
+type SpawnArguments struct {
+	PublicKeys []core.PublicKey
+}
+
+// Signatures is a collections of parts that must satisfy multisig
+// threshold requirement.
+type Signatures []Part
+
+// Part contains a reference to public key and signature from private key counterpart.
+type Part struct {
+	Ref uint8
+	Sig core.Signature
+}
+
+// SpendArguments ...
+type SpendArguments struct {
```

Comment:
SpendArguments and SpendPayload are the same as single sig.
do you think there are downsides to use the same structs?

Reply:
right, will reuse